sig-instrumentation/charter.md

```diff
@@ -0,0 +1,52 @@
+# SIG Instrumentation Charter
+
+This charter adheres to the conventions described in the [Kubernetes Charter README] and uses
+the Roles and Organization Management outlined in [sig-governance].
+
+## Scope
+
+Covers best practices for cluster observability through metrics, logging, and
+events across all Kubernetes components and development of relevant components.
+
+### In scope
+
+Coordinating metric requirements of different SIGs for other components through finding common APIs.
+
+[Link to SIG section in sigs.yaml][sigs.yaml].
+
+#### Code, Binaries and Services
```

Comment:
It would be worthwhile to link to the SIG's sub-projects https://github.com/kubernetes/community/tree/master/sig-instrumentation#subprojects